include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -217,14 +217,14 @@ class argument_parser
      *                     regarded as a container).
      *                     See <a href="https://en.cppreference.com/w/cpp/concept/FormattedInputFunction"> FormattedInputFunction </a>.
      * \tparam validator_type The type of validator to be applied to the option
-     *                        value. Must satisfy seqan3::validator.
+     *                        value. Must model seqan3::validator.
      *
-     * \param[out] value     The variable in which to store the given command line argument.
-     * \param[in]  short_id  The short identifier for the option (e.g. 'a').
-     * \param[in]  long_id   The long identifier for the option (e.g. "age").
-     * \param[in]  desc      The description of the option to be shown in the help page.
-     * \param[in]  spec      Advanced option specification, see seqan3::option_spec.
-     * \param[in]  validator The validator applied to the value after parsing (callable).
+     * \param[in, out] value       The variable in which to store the given command line argument.
+     * \param[in]      short_id    The short identifier for the option (e.g. 'a').
+     * \param[in]      long_id     The long identifier for the option (e.g. "age").
+     * \param[in]      desc        The description of the option to be shown in the help page.
+     * \param[in]      spec        Advanced option specification, see seqan3::option_spec.
+     * \param[in]      verificator A seqan3::validator that verifies the value after parsing (callable).
```

Comment:
didn't we decide to not do this indentation thing anymore? It is not depicted by the generated html data and a single renaming will always destroy it and make it worse. I'd slightly prefer to not do this indentation.

Reply:
We said we don't care how everybody does their documentation :D